core/src/tasks/appRefreshQuery/query.ts

```diff
@@ -18,7 +18,7 @@ export class AppRefreshQueryQuery extends CLSTask {
   }: {
     appRefreshQueryId: string;
   }) {
-    const appRefreshQuery = await AppRefreshQuery.findOne({
+    const appRefreshQuery = await AppRefreshQuery.scope(null).findOne({
```

Comment:
Why does this task want to run non-ready AppRefreshQueries?

Reply:
it should not!